### PR TITLE
Feed avatar

### PR DESCRIPTION
**Pull request by oliamb**
Split each activity summary and display a user popover if this is a HBP User.

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -66,27 +65,22 @@ gulp.task('karma:dist', ['js'], function(done) {
...
+    'src/footer.txt'
   ], {base: 'src'})
   .pipe(plumber())
   .pipe(sourcemaps.init())
-    .pipe(ngAnnotate({single_quotes: true}))
     .pipe(embedTemplates())
     .pipe(concat('angular-hbp-collaboratory.js'))
-    .pipe(hf('(function() {\n"use strict";\n', '})();'))
-    .pipe(hf.header('./LICENSE'))
+    .pipe(ngAnnotate({single_quotes: true}))
```
Comment (oliamb) [inline gulpfile.js]: Change in this file are to fix sourcemaps generation

File: karma.conf.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ module.exports = function(config) {
     ]),
     browsers: ['PhantomJS'],
     frameworks: ['jasmine'],
-    reporters: ['spec', 'coverage'],
+    reporters: ['progress', 'coverage'],
```
Comment (oliamb) [inline karma.conf.js]: A simplier output

File: src/identity/user.service.js [hunk 3]
```diff
@@ -116,7 +118,7 @@ function clbUser(
    * provided list of IDs.
    * @function get
    * @memberof module:clb-identity.clbUser
```
Comment (oliamb) [inline src/identity/user.service.js]: get should retrieve a single result if no array has been passed as input. This preserve backwards compatibility as the method was not accepting single element before. It is more obvious how to use the service with this addition.

File: src/stream/resource-locator.provider.js [hunk 4]
```diff
@@ -41,12 +44,13 @@ function clbResourceLocatorProvider() {
...
- * @param {object} $log AngularJS injection
- * @param {object} clbError AngularJS injection
+ * @param {object} $q        DI
+ * @param {object} $log      DI
+ * @param {object} $injector DI
+ * @param {object} clbError  DI
  * @return {object} the service singleton
  */
-function clbResourceLocator($q, $log, clbError) {
+function clbResourceLocator($q, $log, $injector, clbError) {
```
Comment (oliamb) [inline src/stream/resource-locator.provider.js]: Support for dynamic injection as to be able to provide at runtime module announced at config time.

**Comment by oliamb** [comment]
You can have a look. That's quite a big change. The current UI would look like this in collaboratory frontend.

screen shot 2016-07-06 at 15 31 30

**Comment by zanitete** [comment]
as far as I can tell, lgtm.
I'll have to review the entire resource-locator.provider.js to understand it better.

**Pull request merged**